exercises/practice/high-scores/.meta/config.json

```
@@ -16,7 +16,6 @@
       "create_fixture.sql",
       "data_results.csv",
       "data_scores.csv",
```

```suggestion
      "data_scores.csv"
```
To make configlet CI happy